cowprotocol/services
# [Easy] Make standard quote validity time configurable

# Description

As title, more context in https://cowservices.slack.com/archives/C0361CDD1FZ/p1695992900162259

# Changes
- Instead of using a hardcoded constant, passing in the validity time via a config parameter

## How to test
Start autopilot and/or orderbook with `--standard-offchain-quote-validity-seconds 30` Get quote request, see validity being shorter than usual.

```
curl -H 'content-type: application/json' --data '{"from": "0xc3792470cee7e0d42c2be8e9552bd651766c5178","buyToken": "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48","sellToken": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2","kind": "sell","sellAmountAfterFee": "100000000000000000"}' http://localhost:8080/api/v1/quote
```
